src/bastion/src/children.rs

```diff
@@ -678,56 +867,117 @@ impl Children {
                 Poll::Ready(None) => unreachable!(),
                 Poll::Pending => pending!(),
             }
+
+            #[cfg(feature = "scaling")]
+            self.autoresize_group().await;
         }
     }
 
-    pub(crate) fn launch_elems(&mut self) {
-        debug!("Children({}): Launching elements.", self.id());
+    pub(crate) fn launch_child(&mut self) {
+        let name = self.name();
+        let parent = Parent::children(self.as_ref());
+        let bcast = Broadcast::new(parent, BastionPathElement::Child(BastionId::new()));
 
+        // TODO: clone or ref?
+        let id = bcast.id().clone();
```

Comment:
It will be cloned anyway when we pass to ChildRef. So we can just ref here. Am I wrong?

Reply:
Yet, that's true